Extensions/FileSystem/JsExtension.js

```diff
@@ -381,6 +381,28 @@ module.exports = {
       .setFunctionName('gdjs.fileSystem.deleteFileAsync')
       .setAsyncFunctionName('gdjs.fileSystem.deleteFileAsyncTask');
 
+    extension
+      .addAction(
+        'ReadDirectory',
+        _('Read a directory'),
+        _(
+          'Reads the contents of a directory (all files and sub-directories) and stores them in an array.'
+        ),
+        _('Read the directory _PARAM0_ into _PARAM1_'),
+        _('Windows, Linux, MacOS/Asynchronous'),
+        'JsPlatform/Extensions/filesystem_delete_file32.png',
+        'JsPlatform/Extensions/filesystem_delete_file32.png'
+      )
+      .addParameter('string', _('Directory path'), '', false)
+      .addParameter('scenevar', _('Variable to store the result'), '', true)
```

Comment:
@arthuro555 any chance you can make a change to use the new "variable" parameter, so that it works with any variable? Thanks!

cc @Bouh @D8H